packages/core/src/connectivity/scp-cf/destination/destination-service.ts

```diff
@@ -234,18 +248,19 @@ function callDestinationService(
     destination = addProxyConfigurationInternet(destination);
   }
 
-  if (
-    options.enableCircuitBreaker ||
-    options.enableCircuitBreaker === undefined
-  ) {
-    return getInstanceCircuitBreaker().fire(destination, config);
+  if (options.enableCircuitBreaker) {
+    return getCircuitBreaker().fire(destination, config);
   }
 
   return executeHttpRequest(destination, config);
 }
 
-function getInstanceCircuitBreaker(breaker?: any): any {
-  return typeof breaker === 'undefined'
-    ? new CircuitBreaker(executeHttpRequest, circuitBreakerDefaultOptions)
-    : breaker;
+function getCircuitBreaker(): DestinationCircuitBreaker {
+  if (typeof circuitBreaker === 'undefined') {
```

Comment:
why not `if (circuitBreaker === 'undefined') {`

Reply:
Generally this is the reason: https://stackoverflow.com/questions/2703102/typeof-undefined-vs-null.
But I agree we don't need this "security" here (nor in most other places).

Reply:
Done